src/care/algorithm_decl.h

```diff
@@ -21,10 +21,26 @@
 #include "LLNL_GlobalID.h"
 #endif // CARE_HAVE_LLNL_GLOBALID
 
-#define CARE_MAX(a,b) a > b ? a : b
-#define CARE_MIN(a,b) a < b ? a : b
-
 namespace care {
+
+template <typename T>
+CARE_HOST_DEVICE CARE_INLINE T abs(const T a)
```

Comment:
Should this also use references?

Reply:
The std::abs does not use references, so I copied that.  The behavior is different since this is templated, so I could go either way.

Reply:
Let me know if you guys prefer a reference for the argument

Reply:
Don't worry about it for now.